### PR TITLE
[chromecast] Migrate to bnd build

### PR DESCRIPTION
**Pull request by wborn**
Migrates the Chromecast Binding to the bnd build.

The library manifest forces slf4j 1.8 so I've created https://github.com/vitalidze/chromecast-java-api-v2/pull/109 to fix this.

### CODE REVIEW

**Comment by davidgraeff** [comment]
How about excluding the logging library completely in the dependency section? I have done that with the embedded broker, it also pulled it a fixed version of logging.

File: addons/binding/org.openhab.binding.chromecast.test/src/test/java/org/openhab/binding/chromecast/internal/handler/ChromecastOSGiTest.java [hunk 1]
```diff
@@ -1,70 +0,0 @@
-/**
```
Comment (J-N-K) [inline addons/binding/org.openhab.binding.chromecast.test/src/test/java/org/openhab/binding/chromecast/internal/handler/ChromecastOSGiTest.java]: Do you want to remove this test? If so, please also remove the `itest` directory and its content

**Comment by wborn** [comment]
Because moquette-broker is not a bundle so Karaf feature validation doesn't check if the Import-Package entry of it's manifest can be satisfied. I tried wrapping the bundle and overwriting the Import-Package manifest entry like the last [Wrap Protocol example](https://ops4j1.jira.com/wiki/spaces/paxurl/pages/3833898/Wrap+Protocol) but it doesn't work.

File: addons/binding/org.openhab.binding.chromecast.test/src/test/java/org/openhab/binding/chromecast/internal/handler/ChromecastOSGiTest.java (hunk 1 above)
Comment (wborn) [reply addons/binding/org.openhab.binding.chromecast.test/src/test/java/org/openhab/binding/chromecast/internal/handler/ChromecastOSGiTest.java]: It's was just a placeholder for adding more tests so I've removed it. The file that got added is the .project file of the itests dir itself. Everytime you import all itests without deselecting the parent it gets automatically created/added.

**Comment by J-N-K** [comment]
Do we need api-v2:0.11-SNAPSHOT? 0.10.6 is available on jcenter

**Comment by wborn** [comment]
The whole point is my fix gets released with 0.11.0 so it's now using 0.11.0-SNAPSHOT so I can test my fix.

**Comment by J-N-K** [comment]
Is a timeline available for the release of 0.11.0? Otherweise I would recommend embedding that for now.

**Comment by wborn** [comment]
I asked for that in https://github.com/vitalidze/chromecast-java-api-v2/pull/109#issuecomment-480043850. It'll probably be released before we finish migrating the other add-ons. ;-)

**Comment by J-N-K** [comment]
Let's wait for Jenkins an I'll merge

**Pull request merged**